packages/ui/src/components/Button/defaultTheme.ts

```diff
@@ -0,0 +1,50 @@
+import { createStyling, type Theme } from '../../styler';
+
+import type { BaseProps } from './types';
+
+export const defaultTheme: Theme<'button' | 'content' | 'icon', BaseProps> = {
+    button: createStyling<BaseProps>(
```

Comment:
Btw, I've created a regex to enable Tailwindcss classes within the `createStyling` function, here's a demo:

https://github.com/prezly/theme-kit-js/assets/67554982/09383b93-dd9a-48ce-b57a-f8a0fcbeabd4

Here's the regex:

```json
    "tailwindCSS.experimental.classRegex": [
        ["createStyling(?:<[^<>]*>)?\\(([^)]*)\\)", "[\"'`]([^\"'`]*).*?[\"'`]"]
    ],
```

That's how you add it in VSCode, there's also a way to add it PHPStorm too!

https://cva.style/docs/getting-started/installation#intellisense

Reply:
Noice! 

Works like a charm in PhpStorm:

![image](https://github.com/prezly/theme-kit-js/assets/370680/cf5c42e9-08cd-4aae-980f-72e63c856d83)